components/google_cloud/sources/bigquery-new-row/README.md

```diff
@@ -0,0 +1,61 @@
+# BigQuery New Row Source
+
+## Introduction
+
+This source emits a new event each time a new row is added to a specific table
+in BigQuery. It achieves so by periodically scanning such table for rows with a
+higher index than previous records, hence it is necessary for the target table
+to be indexed by a monotonically increasing column, or at least have a column
+with such a property so that this component can detect new rows.
+
+## Prerequisites
+
+To set up your BigQuery credentials, go to the [Pipedream
+Apps](https://pipedream.com/apps) page, and click on the **CONNECT AN APP**
```

Comment:
The route for this is now https://pipedream.com/accounts

Reply:
OK, I'll fix that. I'll fix other `README`s that refer to the old link as well.